com.unity.ml-agents.extensions/Runtime/Sensors/PhysicsSensorSettings.cs

```diff
@@ -95,51 +94,53 @@ public static int WritePoses(this ObservationWriter writer, PhysicsSensorSetting
             var offset = baseOffset;
             if (settings.UseModelSpace)
             {
-                var poses = poseExtractor.ModelSpacePoses;
-                var vels = poseExtractor.ModelSpaceVelocities;
-
-                for(var i=0; i<poseExtractor.NumPoses; i++)
+                foreach (var pose in poseExtractor.GetModelSpacePoses())
                 {
-                    var pose = poses[i];
-                    if(settings.UseModelSpaceTranslations)
+                    if (settings.UseModelSpaceTranslations)
                     {
                         writer.Add(pose.position, offset);
                         offset += 3;
                     }
+
                     if (settings.UseModelSpaceRotations)
                     {
                         writer.Add(pose.rotation, offset);
                         offset += 4;
                     }
+                }
+
+                foreach(var vel in  poseExtractor.GetModelSpaceVelocities())
```

Comment:
I see that the API changed from IList -> IEnumerable.  Is there a specific reason for that?